pytest_rally/plugin.py

```diff
@@ -62,9 +62,9 @@ def current_branch(repo):
         branches = run_command_with_output(cmd).split("\n")
         current = next(filter(lambda b: b.startswith("*"), branches))
         if "detached" in current:
-            return run_command_with_output(f'git -C {repo} rev-parse HEAD')
+            return run_command_with_output(f'git -C {repo} rev-parse HEAD').strip()
```

Comment:
It's the output of `git rev-parse HEAD` that includes an `\n` and needs to be stripped. I believe this would be clearer:

```suggestion
            return run_command_with_output(f'git -C {repo} rev-parse HEAD').rstrip("\n")
```

If we were not supporting Python 3.8 we could use removesuffix(), available in Python 3.9+.